src/app/components/App/Form.tsx

```diff
@@ -144,6 +160,14 @@ const Form: React.FC = () => {
                   onBlur={handleBlur}
                 />
               </div>
+              <Dropzone setFileSelected={setFileSelected} setImagePreview={setImagePreview}/>
+              <div css={inputStyle}>
+                {imagePreview != undefined && imagePreview != '' ? (
```

Comment:
checking against '' should be enough, as you have initial value as empty string.
but please (always!) use type comparison:
`imagePreview !== ''`

Reply:
yeah that is important especially for numbers as xy != undefined would not work for 0